quickwit/quickwit-serve/src/lib.rs

```diff
@@ -114,6 +115,12 @@ fn has_node_with_metastore_service(members: &[ClusterMember]) -> bool {
 }
 
 pub async fn serve_quickwit(config: QuickwitConfig) -> anyhow::Result<()> {
+    let mut registry = Registry::for_namespace(&config.node_id);
```

Comment:
I may miss something but currently we don't need this registry right? is it part of a hidden plan? :)

Reply:
Yeah, the registry is unnecessary for this PR, but I plan to use it in future works. I can add the registry later and hold the event broker in `QuickwitServices` for the time being.